source/SkiaSharp.Views/SkiaSharp.Views.WPF/SkiaSharp.Views.WPF.csproj

```diff
@@ -9,6 +9,14 @@
     <Title>SkiaSharp Views &amp; Layers for Windows Presentation Foundation (WPF)</Title>
     <PackageTags>wpf</PackageTags>
   </PropertyGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'net462'">
+    <PackageReference Include="OpenTK" Version="3.3.1" NoWarn="NU1701" />
+    <PackageReference Include="OpenTK.GLWpfControl" Version="3.3.0" NoWarn="NU1701" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1'">
```

Comment:
I'm referring to this conditional. I'm going to try != 'net462' instead, but maybe there is a better thing to put here.